src/Controller/EnMarche/DonationController.php

```diff
@@ -140,7 +140,8 @@ public function resultAction(Request $request, Donation $donation, MembershipReg
             'error_code' => $request->query->get('code'),
             'donation' => $donation,
             'retry_url' => $retryUrl,
-            'is_new_adherent' => $request->query->get('is_new_adherent'),
+            'is_registration' => $request->query->get('is_registration'),
+            'is_adherent' => $request->query->get('is_adherent'),
```

Comment:
Why do we need to get the information from the query string while we could have it directly with `$donation` object and a call to the repository? Like https://github.com/EnMarche/en-marche.fr/pull/2420/files#diff-ff29cb5887bec45ec82e598374291962R142